husarion_ugv_docking/config/docking_server.yaml

```
@@ -3,42 +3,48 @@
     use_sim_time: false
     enable_stamped_cmd_vel: true
 
-    controller_frequency: 10.0
+    controller_frequency: 50.0
     initial_perception_timeout: 5.0
     wait_charge_timeout: 10.0
     dock_approach_timeout: 20.0
     undock_linear_tolerance: 0.19
     undock_angular_tolerance: 0.08
     max_retries: 3
-    base_frame: <robot_namespace>/wibotic_receiver_link # Distance to the dock is calculated from this frame
-    fixed_frame: <robot_namespace>/odom
+    base_frame: "<robot_namespace>/wibotic_receiver_link" # Distance to the dock is calculated from this frame
+    fixed_frame: "<robot_namespace>/odom"
```

```suggestion
    base_frame: <robot_namespace>/wibotic_receiver_link # Distance to the dock is calculated from this frame
    fixed_frame: <robot_namespace>/odom
```
Other config keys does not use quotation marks